client/electron/electron-builder.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
There are merge conflicts, you may simplify take all changes from `master` and then update the paths inside the files.

Reply:
It would help if we could see the diff. You can revert, then do the move in one commit, then edit in another.